lib/view_component/system_test_helpers.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module ViewComponent
+  module SystemTestHelpers
+    include TestHelpers
+    include Capybara::DSL
+
+    def page
+      Capybara.current_session
+    end
+
+    def visit_rendered_component_in_browser(component, **options)
+      layout = options[:layout] || false
+
+      opts = {
+        layout: layout,
```

Comment:
Would it be possible to add a configuration option for the default layout to use? It might even be better to allow defining the [controller to use](https://viewcomponent.org/guide/testing.html#configuring-the-controller-used-in-tests) instead so that the layout can be inherited from that.

I made the same comment elsewhere - if you intend on doing so, you may need to update that documentation too.

Reply:
Hmm, I'am not sure tbh. I see its utility of adding a default but haven't yet hit that use case yet. What do you think about holding off on that change until we have a use-case and issue that requests that?